hoppfish/hwangsae-recorder-user-command.py

```diff
@@ -136,7 +144,9 @@ def get(self, method):
         return response, http_ret
 
 api.add_resource(srt, '/api/v1.0/record/<method>/<edge_id>')
-api.add_resource(vod, '/api/v1.0/vod/<method>')
+api.add_resource(vod, '/api/v1.0/vod/<method>/',
+                      '/api/v1.0/vod/<method>/<edge_id>',
+                      '/api/v1.0/vod/<method>/<edge_id>/<file_id>')
 
```

Comment:
Does it working properly? It returns 308(Permanent Redirect’) response.
``` 
> url -G http://localhost:9090/api/v1.0/vod/lookup-by-edge -d 
 "edgeId=0ec04e5f07456d55e3c4794762280f584128d0a3f66f89b73c1792021445b036"

 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
 <title>Redirecting...</title>
 <h1>Redirecting...</h1>
 <p>You should be redirected automatically to target URL: <a href="http://localhost:9090/api/v1.0/vod/lookup-by-edge/?edgeId=0ec04e5f07456d55e3c4794762280f584128d0a3f66f89b73c1792021445b036">http://localhost:9090/api/v1.0/vod/lookup-by-edge/?edgeId=0ec04e5f07456d55e3c4794762280f584128d0a3f66f89b73c1792021445b036</a>.  If not click the link.
```
Also, delete is not GET method. 
I am not sure, but doesn't api.add_resource() be changed something like as below?

```
class vod_url(Resource):
    def get(self, edge_id, file_id):
        try:
            ...
        except Exception as e:
            ...
class vod_delete(Resource):
    def delete(self, edge_id, file_id):
        try:
            ...... 
        except Exception as e:
            ......

api.add_resource(vod, '/api/v1.0/vod/<method>')
api.add_resource(vod_url, '/api/v1.0/vod/url/<edge_id>/<file_id>')
api.add_resource(vod_delete, '/api/v1.0/vod/<edge_id>/<file_id>')
```

Is there another way to test?

Reply:
> Does it working properly? It returns 308(Permanent Redirect’) response

In my tests this solution works, probably the redirection response is masked by my command `curl -L`

> Also, delete is not GET method.

Thanks for point me to this error.

I have changed the code to adapt to your requirements. Thanks for your review.